### ISSUE TITLE
Possible typo in readme.md regarding property bindings

### ISSUE THREAD

**Issue by rsmith31415** [michaelbromley/ng2-pagination#20]
=======

**Angular 2 version**: 2.0.0-beta.15

**ng2-pagination version**: 0.0.1

**Description of issue**: Possible typo in readme.md

**Steps to reproduce**: Currently, the `PaginationControlsCmp` section shows this example:

```
<pagination-controls  id="some_id"
                      (pageChange)="pageChanged($event)"
                      maxSize="9"
                      directionLinks="true"
                      autoHide="true">
</pagination-controls>
```

This [demo](http://plnkr.co/edit/JVQMPvV8z2brCIzdG3N4?p=preview) doesn't seem to be changed when passing additional options. For example, `directionLinks="true"` doesn't remove the directional buttons. However, looking at the code [here](http://michaelbromley.github.io/ng2-pagination/), we need to use property bindings:

```
<pagination-controls [id]="config.id"
                         [maxSize]="maxSize"
                         [directionLinks]="directionLinks"
                         [autoHide]="autoHide"
                         (pageChange)="onPageChange($event)"></pagination-controls>
```

**Demo**: (if possible, edit [this Plunker demo](http://plnkr.co/edit/JVQMPvV8z2brCIzdG3N4?p=preview) and paste the link to your fork)

[This](http://plnkr.co/edit/zwnhiPFbjyoKXb7c56It?p=preview) doesn't seem to work, but [this](http://plnkr.co/edit/Zp8vYdSl8m6Q9MgTRCuw?p=preview) works.

**Comment by michaelbromley** [comment]
Ah - good catch. This is because of the difference between binding and non-binding a boolean. 

When you bind (`[directionLinks]="false"`) the component receives the boolean value `false`. 
When you do not bind (`directionLinks="false"`) the component receives the string value `"false"`.

So in the template, when we evaluate `*ngIf="directionLinks"`, the string "false" is truthy.

The same would also go for the `autoHide` property.

**Comment by michaelbromley** [comment]
Fixed in v0.1.0.

**Comment by rsmith31415** [comment]
Interesting. I didn't know that. Thanks
